drift_limits.ipynb

```diff
@@ -0,0 +1,169 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Maximum dynamical ACA offset size between 2016:180 to 2017:180\n",
+    "\n",
+    "Compute the maximum dynamical ACA offset between T_ccd = -10 C to -17 C.  The\n",
+    "answer is about 37 arcsec for the ACIS-I cycle 18 aimpoint."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from chandra_aca.drift import get_aca_offsets\n",
+    "from astropy.table import Table"
+   ]
```

Comment:
Not sure about our convention for notebooks, but this should probably just import numpy as np too.